Autopilot/PathManager/Inc/landingManager.hpp

```diff
@@ -0,0 +1,63 @@
+#pragma once
+#include "waypointManager.hpp"
+
+const static double DISTANCE_OF_LANDING = 10; //in meters
```

Comment:
To Sahil's point, we should thing about storing these somewhere else, not really sustainable to have them scattered al over the place. 

To my roommate's point, it turns out C++ has something called a constexpr, which guarantees compile time evaluation.